api/services/elastigroup/aws/schemas/itf/update/itfConfigUpdateGroup.yaml

```diff
@@ -0,0 +1,32 @@
+title: Intelligent Traffic Flow configurations
+type: object
+description: |
+  Intelligent Traffic Flow configurations
+properties:
+  fixedTargetGroups:
+    type: boolean
+    description: |
+      Boolean to control whether to keep the set of target groups fixed. When set to true, all target groups will be created in advance upon create/update group, and will not be deleted even when not being used anymore
+    default: false
+    example: false
+  weightStrategy:
+    type: string
+    description: |
+      Distribution strategy (either vCPU or custom weights). Custom strategy can be set only when group has custom weights
```

Comment:
Distribution strategy (either 'vcpu' or 'custom'). 'Custom' strategy can be set only when the group is configured with custom weights.